packages/editor/src/lib/editor/managers/LicenseManager.ts

```diff
@@ -68,6 +93,33 @@ export class LicenseManager {
 		}
 	}
 
+	private getExpirationDateWithGracePeriod(expiryDate: Date) {
+		return new Date(
+			expiryDate.getFullYear(),
+			expiryDate.getMonth(),
+			expiryDate.getDate() + GRACE_PERIOD_DAYS + 1 // Add 1 day to include the expiration day
```

Comment:
i think GRACE_PERIOD_DAYS will be per license but this is fine for now

Reply:
I think @steveruizok said we don't need that per license so I didn't include it. We can always add it if needed.